scripts/build/build_node/Platform/Windows/install-windows-android.ps1

```diff
@@ -21,6 +21,8 @@ Start-Process -FilePath $sdkmanager -ArgumentList $android_packages -NoNewWindow
 Start-Process -FilePath $sdkmanager -ArgumentList $googleplay_packages -NoNewWindow -Wait
 Start-Process -FilePath $sdkmanager -ArgumentList $build_tools -NoNewWindow -Wait
 Start-Process -FilePath $sdkmanager -ArgumentList $ndk -NoNewWindow -Wait
+# Set the NDK environment
+[Environment]::SetEnvironmentVariable("LY_NDK_DIR", "C:\AndroidSdk\ndk\25.1.8937393", [EnvironmentVariableTarget]::Machine)
```

Comment:
Should we check if that NDK path exists? Is this just for running on specific nodes so we already know the exact NDK?

Reply:
No, that is based on `ANDROID_HOME` above, and that path is created automatically by line 23.